MapboxNavigationTests/MapboxVoiceControllerTests.swift

```diff
@@ -45,15 +45,8 @@ class MapboxVoiceControllerTests: XCTestCase {
     func testVoiceDeinit() {
         var voiceController: MockMapboxVoiceController? = MockMapboxVoiceController()
         let deinitExpectation = expectation(description: "Voice Controller should deinitialize")
-        voiceController?.deinitExpectation = deinitExpectation
-        let bundle = Bundle(for: MapboxVoiceControllerTests.self)
-        let filePath = bundle.path(forResource: "turn_left", ofType: "data")
-        let filePathURL = URL(fileURLWithPath: filePath! )
-        let data = try! Data(contentsOf: filePathURL)
-        voiceController?.play(data)
+        voiceController!.deinitExpectation = deinitExpectation
         voiceController = nil
-        XCTAssertNil(voiceController, "voiceController should be nil after nilling it.")
```

Comment:
Is it still possible to keep this assertion?

Reply:
We could but it doesn't validate that there's no leak. voiceController is set to nil on the line before. If it was leaking,  deinit wouldn't get called.

Reply:
right; we need to assign a weak pointer before attempting to deallocate it, similar to https://github.com/mapbox/mapbox-navigation-ios/blob/8e094d08a2bbe05ace870f52006e8d02f48dc228/MapboxNavigationTests/LeakTest.swift#L17 or https://github.com/mapbox/mapbox-gl-native/blob/c5625f988a37b6876c4ef5fc8d9de295ee28ed44/platform/ios/Integration%20Tests/MBGLIntegrationTests.m#L15